### PR TITLE
Update Microsoft.NETCore.Sdk to 1.0.0-alpha-20160831-1

### PR DESCRIPTION
**Pull request by eerhardt**
@brthor @livarcocc @Sridhar-MS

### CODE REVIEW

File: src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-using System.Text;
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 namespace Microsoft.DotNet.ProjectJsonMigration
 {
     public class ConstantPackageNames
     {
-        public const string CSdkPackageName = "Microsoft.DotNet.Core.Sdk";
+        public const string CSdkPackageName = "Microsoft.NETCore.Sdk";
```
Comment (brthor) [inline src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs]: 👍

**Comment by brthor** [comment]
:shipit:

**Comment by eerhardt** [comment]
@brthor - can you take a quick look at my last commit?  The latest Sdk fixed some tests.

File: test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs [hunk 2]
```diff
@@ -56,19 +56,18 @@ public void It_migrates_dotnet_new_console_with_identical_outputs()
         }
 
         [Fact]
-        // Outputs Not Identical: https://github.com/dotnet/sdk/issues/97
         public void It_migrates_dotnet_new_web_with_outputs_containing_project_json_outputs()
```
Comment (brthor) [inline test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs]: very stoked to see this :+1:

**Comment by brthor** [comment]
LGTM

**Comment by eerhardt** [comment]
test OSX x64 Release Build

**Pull request merged**